ods_ci/tests/Tests/400__ods_dashboard/415__ods_dashboard_projects/415__ods_dashboard_projects.robot

```diff
@@ -395,6 +429,49 @@ Verify User Can Create A Workbench With Environment Variables
     ...    auth_type=${TEST_USER_3.AUTH_TYPE}
     Environment Variables Should Be Available In Jupyter    exp_env_variables=${envs_list}
 
+Verify User Can Edit Name and Description In A Workbench
+    [Tags]    Sanity    Tier1    ODS-1931
+    [Documentation]    Verifies users can edit name and description fields in an existing workbench
+    [Setup]   Launch Data Science Project Main Page
+    [Teardown]    Delete Data Science Project    project_title=${PRJ_TITLE1}
+    Open Data Science Projects Home Page
+    Create Data Science Project    title=${PRJ_TITLE1}    description=${PRJ_DESCRIPTION}
+    ...    resource_name=${NONE}
+    Open Data Science Project Details Page       project_title=${PRJ_TITLE1}
+    Create Workbench    workbench_title=${WORKBENCH_4_TITLE}  workbench_description=${WORKBENCH_DESCRIPTION}
+    ...                 prj_title=${PRJ_TITLE1}    image_name=${NB_IMAGE}   deployment_size=Small
+    ...                 storage=Persistent  pv_name=${WORKBENCH_4_TITLE}-PV  pv_existent=${FALSE}
+    ...                 pv_description=${NONE}  pv_size=${2}
+    ...                 press_cancel=${FALSE}
+    Wait Until Workbench Is Started     workbench_title=${WORKBENCH_4_TITLE}
+    Edit Workbench    workbench_title=${WORKBENCH_4_TITLE}    workbench_title_update=${WORKBENCH_4_TITLE}Updated
+    ...               workbench_description=Latest${WORKBENCH_DESCRIPTION}
+    Sleep    5    reason=Waiting for Workbench Updates to be complete
+    Verify Workbench Is Edited    workbench_title=${WORKBENCH_4_TITLE}Updated    deployment_size=Small
+    ...                workbench_description=Latest${WORKBENCH_DESCRIPTION}     image_name=${NB_IMAGE}
+
+Verify User Can Edit A Environment Variables In An Existing Workbench
+    [Tags]    Sanity    Tier1    ODS-2272
+    [Documentation]    Verifies users can edit an environment variable within an existing workbench
+    [Setup]   Launch Data Science Project Main Page
+    [Teardown]    Run Keywords      Delete Workbench From CLI    workbench_title=${WORKBENCH_4_TITLE}    project_title=${PRJ_TITLE2}
+    ...    AND     Delete Data Science Project     project_title=${PRJ_TITLE2}
```

Comment:
you can delete the project directly with the kw to delete projects via CLI. That would spare some seconds